stable/prometheus-adapter/values.yaml

```diff
@@ -51,6 +51,17 @@ rules:
   # Mounts a configMap with pre-generated rules for use. Overrides the
   # default and custom entries
   existing:
+externalRules:
+  custom: []
```

Comment:
What about instead of `externalRules.custom: []` make it `rules.external: []`? Seems cleaner to me.

Reply:
Done